pkg/virtualnetwork/ssh_forwarder.go

```diff
@@ -0,0 +1,229 @@
+package virtualnetwork
```

Comment:
Maybe we can keep this outside of pkg/ for the moment and keep it in cmd/gvproxy ?

I recognize cmd/gvproxy is growing a lot and it can deserve a better split. 
pkg/virtualnetwork is vendored by crc also and they don't need this. So I guess it's better to not include it here.

Reply:
Ok makes sense. It does have some slight internal member usage IIRC, but can probably be worked around with minor enhancements to this package.